IFAS/DAL/HistoryDAL.py

```diff
@@ -0,0 +1,14 @@
+from db import db_connection
+from sqlalchemy import text
+
+class historyData:
```

Comment:
Do we really need another class for history? Don't you think we just get the history from the if's table? fix that - it is a duplication right now

Reply:
yes i think we need 2 tables, one for history and one for the ifs, as mention in the exercise